src/components/ReportActionItem/IOUPreview.js

```diff
@@ -240,20 +228,7 @@ const IOUPreview = (props) => {
                         <Text style={[styles.textLabel, styles.colorMuted]}>{props.translate('iou.pendingConversionMessage')}</Text>
                     )}
 
-                    <Text style={[styles.colorMuted]}>{Str.htmlDecode(lodashGet(props.action, 'originalMessage.comment', ''))}</Text>
-
-                    {isCurrentUserManager && !props.shouldHidePayButton && props.iouReport.stateNum === CONST.REPORT.STATE_NUM.PROCESSING && (
-                        <Button
-                            style={styles.mt4}
-                            onPress={props.onPayButtonPressed}
-                            onPressIn={() => DeviceCapabilities.canUseTouchScreen() && ControlSelection.block()}
-                            onPressOut={() => ControlSelection.unblock()}
-                            onLongPress={showContextMenu}
-                            text={props.translate('iou.pay')}
-                            success
-                            medium
-                        />
-                    )}
+                    <Text style={[styles.colorMuted]}>{Str.htmlDecode(lodashGet(props.action, 'originalMessage.comment', requestCurrency))}</Text>
```

Comment:
why are we showing the comment in one case but the currency in another?

looking at the props.action locally looks like there should be currency available to us in `originalMessage`:
<img width="1512" alt="image" src="https://github.com/Expensify/App/assets/4073354/0a4f5064-252a-4a01-8893-1653b10e0cfe">

Reply:
nice catch. removed!